### DISCUSSION TITLE
Unable to reapply for student developer pack

### DISCUSSION THREAD

**Discussion by oshi36** [community/community#17714] [GitHub Education]
I am a student graduating in 2022 and submitting my PR for GitHub graduation 2022 and reapplying for the student developer pack.
As I don't have a dated school ID so I tried submitting a letterhead of my university which shows my enrollment but still it's not accepting.I tried it many times earlier also and tried to submit a screenshot of my college portal which shows my current enrollment. 
But still not getting it accepted.
Any help would be appreciated.

**Comment by elisemoe** [comment]
Thanks for letting us know @oshi36! Please [link to your PR in this Issue](https://github.com/education/GitHubGraduation-2022/issues/4290) and we'll manually review your submissions.

**Comment by oshi36** [comment]
Hi @elisemoe , I have linked my [PR to issue you mentioned above](https://github.com/education/GitHubGraduation-2022/issues/4290#issuecomment-1126655740). 
Any updates on this?
Thank you.
